src/components/HelpModal.js

```diff
@@ -0,0 +1,22 @@
+import ReactModal from 'react-modal'
+import '../css/HelpModal.css'
+
+function HelpModal({ isOpen, close}) {
+    return (
+        <ReactModal className="modal" isOpen={isOpen} parentSelector={() => document.querySelector('.Game')} contentLabel="Instructions">
+            <div className='help'>
+                <button className="close-btn" onClick={close}>🆇</button>
+                <h3>How to Play:</h3>
+                <p className="instructions">1. Select <span>"Set Game Board!"</span> at the bottom of the screen. You'll see country names appear in the game board squares.</p>
```

Comment:
Do we want "Click" instead of "Select" in these lines? I know that technically, those buttons can probably be used with the keyboard, but if we have middle-schoolers using this app, "Select" might actually leave them unclear as to which action to take.

Reply:
Also, "country names and flags" rather than "country names" is probably more specific.

Reply:
Implemented!